packages/eas-cli/src/update/utils.ts

```diff
@@ -201,8 +201,8 @@ export function formatUpdateTitle(update: UpdateFragment): string {
   )} by ${actorName}, runtimeVersion: ${runtimeVersion}] ${message}`;
 }
 
-export function getUpdateGroupJsonInfo(updateGroups: UpdateFragment[]): UpdateJsonInfo[] {
-  return updateGroups.map(update => ({
+export function getUpdatesJsonInfo(updates: UpdateFragment[]): UpdateJsonInfo[] {
```

Comment:
optional, though I find it clearer

```suggestion
export function getUpdateJsonInfosForUpdates(updates: UpdateFragment[]): UpdateJsonInfo[] {
```